lib/kitchen/directions/bake_references/v3.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeReferences
+  class V3
+
+    def bake(book:)
+      book.chapters.each do |chapter|
+        chapter.pages.each do |page|
+          page.references.each do |reference|
+            title = reference.titles.first
+            title.name = 'h2'
+            title['data-type'] = 'document-title'
+            title['id'] = page.title.copied_id
+            title.replace_children(with:
+              <<~HTML.chomp
+                <span class="os-number">#{page.title.children[0].text}</span>
+                <span class="os-divider"> </span>
+                <span class="os-text" data-type="" itemprop="">#{page.title_text} </span>
+              HTML
+            )
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
I'm wondering if we can create something like that https://github.com/openstax/kitchen/blob/b36c33685e89c8b797d6a6fee6dbf1953725d43d/lib/kitchen/directions/eoc_section_title_link_snippet.rb for this code :thinking: @kswanson33 you were working on that. What do you think?

Reply:
agreed! it looks like it could be the same thing -- does it work to substitute in `EocSectionTitleLinkSnippet` here?

Reply:
I actually don't use links in these titles, so I don't think I can use that snippet without modifying it